pandas/tests/io/formats/test_format.py

```diff
@@ -2181,18 +2181,18 @@ def test_to_string_float_na_spacing(self):
         result = s.to_string()
         expected = (
             "0       NaN\n"
-            + "1    1.5678\n"
-            + "2       NaN\n"
-            + "3   -3.0000\n"
-            + "4       NaN"
+            "1    1.5678\n"
+            "2       NaN\n"
+            "3   -3.0000\n"
+            "4       NaN"
         )
         assert result == expected
 
     def test_to_string_without_index(self):
         # GH 11729 Test index=False option
         s = Series([1, 2, 3, 4])
         result = s.to_string(index=False)
-        expected = "1\n" + "2\n" + "3\n" + "4"
+        expected = "".join(["1\n", "2\n", "3\n", "4"])
```

Comment:
Could probably be simplified as either `"1\n2\n3\n4"` or `"\n".join(["1", "2", "3"])`

(similar for the next change)

Reply:
Thank you, I did as you suggested. It works very well.